lib/database_validations/rubocop/cop/belongs_to.rb

```diff
@@ -0,0 +1,46 @@
+module RuboCop
+  module Cop
+    module DatabaseValidations
+      # Use `db_belongs`_to instead of `belongs_to`.
+      #
+      # @example
+      #   # bad
+      #   belongs_to :company
+      #
+      #   # good
+      #   db_belongs_to :company
+      #
+      class BelongsTo < Cop
+        MSG = 'Use `db_belongs_to`.'.freeze
+
+        NON_SUPPORTED_OPTIONS = %i[
```

Comment:
it might be better to list the supported ones in order to avoid false positives in the future. WDYT?

Reply:
I agree with we use the whitelist approach. It would be good if we can mix metaprogramming and get it from the method arguments that are actually supporting.